snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/compare/ReferenceSetCompare.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2020 B2i Healthcare. All rights reserved.
+ *******************************************************************************/
+package com.b2international.snowowl.snomed.datastore.compare;
+
+import com.b2international.commons.exceptions.BadRequestException;
+import com.b2international.snowowl.core.compare.CompareSetResult;
+import com.b2international.snowowl.core.compare.CompareSets;
+import com.b2international.snowowl.snomed.common.SnomedRf2Headers;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedRefSetType;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedReferenceSet;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedReferenceSetMember;
+
+/**
+ * @since 7.8
+ */
+public final class ReferenceSetCompare implements CompareSets<SnomedReferenceSetMember, SnomedReferenceSet> {
+
+	@Override
+	public CompareSetResult<SnomedReferenceSetMember> doCompare(SnomedReferenceSet baseRf, SnomedReferenceSet compareRf) {
+		if (!SnomedRefSetType.SIMPLE_MAP.equals(baseRf.getType()) || !SnomedRefSetType.SIMPLE_MAP.equals(compareRf.getType())) {
```

Comment:
Too complex boolean expression, invert it to make it more readable.

Reply:
Done